.github/workflows/cord-ci.yml

```diff
@@ -0,0 +1,47 @@
+name: cord-ci
+on:
+  pull_request:
+    branches: [ develop, dev-0.8.0]
+    paths-ignore:
+      - "doc/**"
+      - "**.md"
+  push:
+    branches: [ develop, dev-0.8.0]
+    paths-ignore:
+      - "doc/**"
+      - "**.md"
```

Comment:
can we separate out the `push` to a separate file? that way, we know which one runs. And do we need a same test to run again?

Reply:
Done.